go/test/endtoend/vault/vault_test.go

```diff
@@ -267,6 +267,12 @@ func initializeClusterLate(t *testing.T) {
 		require.NoError(t, err)
 	}
 
+	for _, tablet := range shard.Vttablets {
```

Comment:
This is done because in vault test we alter user passwords before we Initialize VTTablet. So I have to set their read_only to `OFF`